main.tex

```diff
@@ -43,6 +47,8 @@
 \newenvironment{issue}{\color{red}}{\color{black}}
 \newcommand{\what}{\textbf{\issue{??}}}
 
+\newcommand{\skipline}{\leavevmode\vspace{\baselineskip}} 
```

Comment:
What's this for?

Reply:
Just to add empty lines, but I removed it.